mdal/frmts/mdal_netcdf.cpp

```diff
@@ -34,7 +34,7 @@ void NetCDFFile::openFile( const std::string &fileName )
   int res = nc_open( fileName.c_str(), NC_NOWRITE, &mNcid );
   if ( res != NC_NOERR )
   {
-    MDAL::debug( nc_strerror( res ) );
+    MDAL::Log::info( nc_strerror( res ) );
```

Comment:
debug.. or cannot we throw LogError struct

Reply:
Let's do it